### PR TITLE
Juniper 2022-04 CVE publications.

### PR DESCRIPTION
**Pull request by ViperGeek**

### CODE REVIEW

File: 2022/22xxx/CVE-2022-22183.json [hunk 1]
```diff
@@ -1,18 +1,165 @@
 {
```
Comment (cve-team) [inline 2022/22xxx/CVE-2022-22183.json:1]: Invalid JSON file (invalid state)

File: 2022/22xxx/CVE-2022-22195.json [hunk 2]
```diff
@@ -1,18 +1,142 @@
 {
```
Comment (cve-team) [inline 2022/22xxx/CVE-2022-22195.json:1]: 'version_value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/22xxx/CVE-2022-22195.json (hunk 2 above)
Comment (ViperGeek) [reply 2022/22xxx/CVE-2022-22195.json:1]: CVE-2022-22195 updated.

File: 2022/22xxx/CVE-2022-22183.json (hunk 1 above)
Comment (cve-team) [inline 2022/22xxx/CVE-2022-22183.json:1]: Invalid JSON file (invalid state)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/22xxx/CVE-2022-22183.json (hunk 1 above)
Comment (ViperGeek) [reply 2022/22xxx/CVE-2022-22183.json:1]: Sorry for missing that.  "STATE": "PUBLIC"

**Pull request merged**